extension/src/experiments/model/index.ts

```diff
@@ -365,6 +365,14 @@ export class ExperimentsModel extends ModelWithPersistence {
     return [...this.getWorkspaceAndCommits(), ...this.getExperiments()]
   }
 
+  public getUnfilteredCommitsAndExperiments() {
```

Comment:
I didn't see a function that already collected all unfiltered experiments/commits but I might have missed something.

Reply:
This will give slightly different results to what is shown in the table as we have to show commits that have > 0 unfiltered experiments

Reply:
Yes, I figured we didn't need to show those commits since the plots data isn't hierarchical like the table. But if we want it to reflect the table exactly, I can update the function?

Reply:
The current implementation is fine.